src/metadataGeneration/methodGenerator.ts

```diff
@@ -120,7 +120,7 @@ export class MethodGenerator {
       if (expression.arguments.length > 1 && (expression.arguments[1] as any).text) {
         description = (expression.arguments[1] as any).text;
       }
-      if (expression.arguments.length > 2 && (expression.arguments[2] as any).text) {
+      if (expression.arguments.length > 2 && expression.arguments[2]) {
```

Comment:
what exactly does this fix?

Reply:
This part of the code is where the example is picked from the _@Response_ decorator.
For example:
```
@Response(400, "If something goes wrong in the request", {errorCode:1209, message:"you sent a bad parameter"})
```

I believe the problem with the previous code is that it was treated like a character string out of the _text_ attribute of an expression object, which it is not at this point. It really looks more like an _any_ typed structure that has of course not the said _text_ attribute, hence the removal of the check.

Removing this tiny extra condition magically makes the example to appear in the swagger documentation again.